src/bump.ts

```diff
@@ -897,6 +901,12 @@ export async function bumpSdkVer(
     }
   }
   core.setOutput("next-version", nextVersion?.toString() ?? "");
+  const latestDraftName = latestDraft?.name ?? "";
+  core.setOutput(
+    "previous-commit",
```

Comment:
This isn't really a previous commit. When a draft gets promoted to an RC, the commit doesn't change. Improvements on the name welcome.